modules/custom/reservations/templates/reservations.html.twig

```diff
@@ -31,6 +31,12 @@
       }
       {% endif %}
     </ul>
+
+     <form id="formName" name="formName">
```

Comment:
formName meni vise zvuci kao da je to ime neke forme, a ne forma koja ima ime. Mozemo li malo drugacije nazvati?